docs/src/main/asciidoc/_cloud/Getting_Started.adoc

```diff
@@ -0,0 +1,156 @@
+
+First, we need to provision a database instance, in this case PostgresSQL:
+```
+$ oc new-app postgresql-ephemeral \
+   -p DATABASE_SERVICE_NAME=todos-db \
+   -p POSTGRESQL_DATABASE=todos
+
+```
+
+Next we use the WildFly Helm Charts to build and deploy our quickstart:
+```
+$ helm install todo-backend --set build.ref={WildFlyQuickStartRepoTag} -f https://raw.githubusercontent.com/wildfly/wildfly-charts/main/examples/todo-backend/todo-backend-bootable-jar.yaml wildfly/wildfly
+```
+
+`build.ref` is used to indicate which quickstart tag is to be used.
+
+Take a look at the `bootable-jar-openshift` profile configuration in the todo-backend quickstart's https://github.com/wildfly/quickstart/blob/main/todo-backend/pom.xml#L254-L304[pom.xml]. The Maven profile named `bootable-jar-openshift` is used by the Helm chart to provision the server with the quickstart deployed. Notice the `wildfly-maven-plugin` configuration defined in this profile. It specifies the layer that should be used when provisioning WildFly. For more details about this, take a look at the https://github.com/wildfly/quickstart/tree/main/todo-backend#architecture[Architecture] section in the quickstart's README.
+
+We have named this application `todo-backend`. This may be changed to a different name, if desired.
+
+The application will now begin to build, and can be observed via:
+```
+$ oc get build -w
+```
+Once the build is complete, deployment will begin:
+```
+$ oc get deployment todo-backend -w
+```
+Once deployment is complete, we can test the application:
+```
+$ oc get route todo-backend -o jsonpath="{.spec.host}"
+```
+Expected output:
+```
+todo-backend-someuser-dev.apps.sandbox.x8i5.p1.openshiftapps.com
+```
+You should now be able to access your application using, for example
+```
+https://todo-backend-someuser-dev.apps.sandbox.x8i5.p1.openshiftapps.com
+```
+You can use the todo-frontend to test out the deployed app using:
+```
+https://todobackend.com/client/index.html
+```
+and entering your application URL from above.
+
+If you wish to remove the application when you are done with it, simply run:
+```
+$ helm delete todo-backend
+```
+
+## Example: jaxrs-client
+The second example to build and deploy to OpenShift is the
+https://github.com/wildfly/quickstart/tree/main/jaxrs-client[jaxrs-client] quickstart.
+
+First of all, clone the quickstart repo:
+```
+$ git clone https://github.com/wildfly/quickstart.git
+cd quickstart/jaxrs-client
+```
+
+WildFly Helm Charts is used to build and deploy `jaxrs-client`. The following command installs a Helm Chart in a OpenShift cluster (users can connect to their sandbox following the instructions already provided above):
```

Comment:
> The following command installs a Helm Chart

To be more accurate, this should be "The following command installs a Helm Release from the WildFly Helm Chart"